.cirrus.yml

```diff
@@ -5,12 +5,12 @@
 # See https://confluence.atlassian.com/bitbucket/yaml-anchors-960154027.html
 
 windows_shard_template: &WINDOWS_SHARD_TEMPLATE
-  only_if: "changesInclude('.cirrus.yml', 'dev/**', 'bin/**') || $CIRRUS_PR == ''" # https://github.com/flutter/flutter/issues/41941
+  only_if: "changesInclude('.cirrus.yml', 'dev/**', 'bin/**') && $CIRRUS_PR == ''" # https://github.com/flutter/flutter/issues/41941
   script:
     - dart --enable-asserts ./dev/bots/test.dart
 
 macos_shard_template: &MACOS_SHARD_TEMPLATE
-  only_if: "changesInclude('.cirrus.yml', 'dev/**', 'bin/**') || $CIRRUS_PR == ''" # https://github.com/flutter/flutter/issues/41941
+  only_if: "changesInclude('.cirrus.yml', 'dev/**', 'bin/**') && $CIRRUS_PR == ''" # https://github.com/flutter/flutter/issues/41941
```

Comment:
I don't see hostonly_devicelab_tests for macOS on any builder, am I missing it?

Reply:
This was an error, back to ||